checkrebuild

```diff
@@ -22,17 +22,26 @@ get_unofficial_pkgs() {
     comm -23 <(pacman -Qq | sort) <(pacman -Sql "${repos_to_skip[@]}" 2>/dev/null | sort)
 }
 
-filter_executable() {
-    while read -r file; do
-        [[ -f "$file" && -x "$file" ]] && echo "$file"
-    done
+newline2null(){
+    perl -pe 's/\n/\0/'
 }
 
 get_package_files() {
     xargs pacman -Qql |
-    perl -pe 's/\n/\0/' |
-    xargs -0 readlink -f |
-    filter_executable
+    newline2null |
+    xargs -0 readlink -e |
+    sort -u
+}
+
+filter_executable() {
+    newline2null |
+    LANG=C xargs -0 stat --printf "%F %a\t%n\n" |
```

Comment:
For my learning, what is the purpose of `LANG=C` here?

Reply:
`stat "%F"` message depend on user locale setting.
So we have to force default local to be able to fileter
for `regular file`